include/vctr/Containers/VctrBase.h

```diff
@@ -366,6 +366,24 @@ class VctrBase : private Config,
     /** Subtracts a constant from this in place */
     void operator-= (value_type c);
 
+    //==============================================================================
+    // Math reduction operations
+    //==============================================================================
+    /** Returns the minimal value of all elements */
+    ElementType min() const requires std::totally_ordered<ElementType>;
```

Comment:
Please terminate doxygen comments with a period for good style. :)

Reply:
Aaah, I already tried to add a lot of them 😄 Will do!

Reply:
I believe you should be able to locate all of them with this regex: `[^. ]{1} *\*/`